lua/orgmode/org/autocompletion/omni.lua

```diff
@@ -38,6 +53,9 @@ local tags = {
 local filetags = {
   line_rgx = vim.regex([[\c^\#+FILETAGS:\s\+]]),
   rgx = vim.regex([[:\([0-9A-Za-z_%@\#]*\)$]]),
+  extra_cond = function(line, _)
+    return not string.find(line, 'file:$')
```

Comment:
Update `file:$` to `file:.*$`, since `filetags` overtakes the context after `file:` is typed.